tests/test_calculate_instance_segmentation_map.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+from parameterized import parameterized
+
+from monai.apps.pathology.transforms.post.array import CalculateInstanceSegmentationMap
+from monai.utils import min_version, optional_import
+from tests.utils import TEST_NDARRAYS
+
+_, has_skimage = optional_import("skimage", "0.19.3", min_version)
+
+np.random.RandomState(123)
+
+TESTS = []
+params = {"connectivity": 1}
+for p in TEST_NDARRAYS:
+    image = p(np.random.rand(1, 10, 10))
+    mask = p((np.random.rand(1, 10, 10) > 0.5).astype(np.uint8))
+    marker = p((np.random.rand(1, 10, 10) > 0.5).astype(np.uint8))
+
+    TESTS.append([params, image, mask, marker, (1, 10, 10)])
+
+ERROR_TESTS = []
+for p in TEST_NDARRAYS:
+    image = p(np.random.rand(3, 10, 10))
+    mask = p((np.random.rand(2, 10, 10) > 0.5).astype(np.uint8))
+    marker = p((np.random.rand(2, 10, 10) > 0.5).astype(np.uint8))
+
+    ERROR_TESTS.append([params, image, mask, marker])
+
+
+@unittest.skipUnless(has_skimage, "Requires scikit-image library.")
+class TestCalculateInstanceSegmentationMap(unittest.TestCase):
```

Comment:
I think other transforms generate data for this transform, is it possible to add unit test to cover the overall case with other transforms?

Thanks.